reporter/src/main/kotlin/reporters/NoticeOnlyDeclaredReporter.kt

```diff
@@ -36,7 +36,7 @@ class NoticeOnlyDeclaredReporter : AbstractNoticeReporter() {
         val licenseFindings = sortedMapOf<String, SortedSet<String>>()
```

Comment:
commit message nit: please add an 's' suffix to 'handle', see '...which handle packages...'

Reply:
Done.